zebra-test/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-test"
-version = "1.0.0-alpha.13"
+version = "1.0.0-alpha.17"
```

Comment:
This should just go to 1.0.0-alpha.14 until we decide what to do about version numbers

Reply:
Got it, changed in c55eacd